### PR TITLE
chore: [CCM-5432]: Fixing Breadcrumbs across CE

### PR DESCRIPTION
**Pull request by jenil-harness**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by jenil-harness** [comment]
retrigger titlecheck

File: src/modules/10-common/navigation/MainNav/MainNav.tsx [hunk 1]
```diff
@@ -101,7 +101,7 @@ export default function L1Nav(): React.ReactElement {
         )}
         {CENG_ENABLED && (
           <li className={css.navItem}>
-            <Link {...commonLinkProps} to={paths.toCEOverview(params)}>
+            <Link {...commonLinkProps} to={paths.toCE(params)}>
```
Comment (rishik30) [inline src/modules/10-common/navigation/MainNav/MainNav.tsx:104]: why are we passing params here if it's not being used in the method?

File: src/modules/75-ce/RouteDestinations.tsx [hunk 2]
```diff
@@ -132,20 +145,23 @@ const CERoutes: React.FC = () => {
         >
           <CEHomePage />
         </RouteWithLayout>
+        <RouteWithLayout licenseRedirectData={licenseRedirectData} path={routes.toCE({ ...accountPathProps })} exact>
```
Comment (rishik30) [inline src/modules/75-ce/RouteDestinations.tsx:148]: why are we passing params to `routes.toCE` method here if it's not being used in the method?

Comment (jenil-harness) [reply src/modules/75-ce/RouteDestinations.tsx:148]: accountPathProps are still required to generate toCE url.

**Review by vkbansal — APPROVED** [review]

**Review by rishik30 — APPROVED** [review]

**Pull request merged**